docs/ADL2/master07.11-adl_rulesNEW.adoc

```diff
@@ -12,6 +12,33 @@ Since multiple data instances may conform to a single archetype structure or sub
 
 The symbols used in rule expressions are defined in two sub-sections under a dedicated archetype `symbols` section: `symbol_definitions` (optional) and `symbol_bindings` (mandatory). The definitions are of the same form as the archetype term definitions and serve the same purpose, i.e. to provide multi-lingual natural language definitions that may be used in tool user interfaces. In each term, a `symbol` attribute may be used to define a translation of the symbol itseld, which may be used They are optional since they are not needed to perform either path mapping or runtime execution.
 
+[tabs,sync-group-id=adl-example]
```

Comment:
Why is this file named adl_rulesNEW.adoc? Why wasn't the original rules file modified?

Reply:
The *NEW file is not used, but for the moment we will leave it in case it is still used somewhere...